components/speedreader/rust/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Generated by Cargo
+# will have compiled files and executables
+target/
```

Comment:
in an ideal world everything generated should go to out/ ...

Reply:
this is to be able to run utilities and examples that are not part of full build, GN-driven build does not output stuff to this `target/`

Reply:
gotcha